runtime/doc/dev_style.txt

```diff
@@ -859,6 +859,25 @@ execute, simply `assert`: >c
         assert(false);
     }
 
+Switch statements that are conditional on an enumerated value may not have a
+`default` case if it is exhaustive. In cases where an exhaustive switch
+statement on an enum is exhaustive, the compiler may still treat it as if it
+is not exhaustive. This happens in cases where each case contains a return
+statement, but the compiler still complains about there not being a return
+statement for the entire control flow. For situations like these, you may use
+`UNREACHABLE` to explicitly tell the compiler that the default case is
+unreachable. For example: >c
+
+    enum { VarFirstCase, VarSecondCase } var;
+    // do some stuff with var
```

Comment:
> Is this comment even necessary? "Do stuff"?

Otherwise it implies that we're using a `switch` on an unset var. Maybe it's better to just set `var` to something but then it looks kinda dumb

Reply:
Oh, you mean that was supposed be "here is code" instead of a comment. In the other sections of the style guide it uses

```

        ...
```

to denote a section of code.